src/annotator/anchoring/pdf.js

```diff
@@ -401,33 +410,36 @@ async function anchorQuote(quoteSelector, positionHint) {
   }
 
   // Search pages for the best match, ignoring whitespace differences.
-  const [strippedPrefix] =
-    quoteSelector.prefix !== undefined ? stripSpaces(quoteSelector.prefix) : [];
-  const [strippedSuffix] =
-    quoteSelector.suffix !== undefined ? stripSpaces(quoteSelector.suffix) : [];
-  const [strippedQuote] = stripSpaces(quoteSelector.exact);
+  const strippedPrefix =
+    quoteSelector.prefix !== undefined
+      ? stripSpaces(quoteSelector.prefix)
+      : undefined;
+  const strippedSuffix =
+    quoteSelector.suffix !== undefined
+      ? stripSpaces(quoteSelector.suffix)
+      : undefined;
+  const strippedQuote = stripSpaces(quoteSelector.exact);
 
   let bestMatch;
   for (let page of pageIndexes) {
     const text = await getPageTextContent(page);
-    const [strippedText, offsets] = stripSpaces(text);
+    const strippedText = stripSpaces(text);
 
     // Determine expected offset of quote in current page based on position hint.
     let strippedHint;
     if (expectedPageIndex !== undefined && expectedOffsetInPage !== undefined) {
-      let hint;
       if (page < expectedPageIndex) {
-        hint = text.length; // Prefer matches closer to end of page.
+        strippedHint = strippedText.length; // Prefer matches closer to end of page.
       } else if (page === expectedPageIndex) {
-        hint = expectedOffsetInPage;
+        [strippedHint] = translateOffsets(
+          text,
+          strippedText,
+          expectedOffsetInPage,
+          expectedOffsetInPage,
+          isNotSpace
+        );
```

Comment:
> When passing the same value for start and end to translateOffsets, is it to be expected that it will return [startOffset, endOffset] where startOffset === endOffset in all cases?

Yes. In this case the number of non-ignored characters between the start and end offsets in the input is zero, and `translateOffsets` will always return a `[start, end]` tuple with `start` === `end` in that scenario.